docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -473,9 +493,11 @@ Content-Type: text/plain;charset=UTF-8
 user%
 ----
 
-== Security JPA Reference Guide 
+:sectnums!:
 
-Now that you have run and tested the demo, please have a look at the more specific information about preparing your JPA identity store.
+== JPA security information 
```

Comment:
@sberyozkin .. does this belong in this tutorial, or should it be somewhere else (howto)? feels out of place in this tutorial.

Reply:
@ebullient Right, originally this document was a `security-jpa.adoc`, quickstart + reference, before it was refactored into a Getting Started with Quarkus Sec. So I guess this JPA security information is a reference part specific to JPA only and should eventually be moved to `security-jpa-reference.adoc` (probably makes sense to do it in the next step)

Reply:
@michelle-purcell Hi Michelle, `JPA security information`  changes the original meaning of `Security JPA Reference Guide`. This is a reference part related to the `security-jpa` extension which is located in this doc because the original doc was all about `security-jpa`. Please retain the original title and then we'll move it to its own ref/etc doc and name accordingly.

Reply:
@ebullient & @sberyozkin - I added a reminder to move this section later in a hidden comment.